doc/flow_risks.rst

```diff
@@ -272,4 +272,9 @@ NDPI_HTTP_CRAWLER_BOT
 ===================================
 The risk is set whenever a crawler/bot/robot has been detected
 
+.. _Risk 045:
 
+NDPI_ANONYMOUS_USER
```

Comment:
`NDPI_ANONYMOUS_USER` may be misleading.
Is `NDPI_ANONYMOUS_ADDRESS` or `NDPI_ANONYMOUS_SUBSCRIBER` more appropriate?

Reply:
`NDPI_ANONYMOUS_SUBSCRIBER` is a better name. Thanks for the suggestion